.github/workflows/md_link_check.yaml

```diff
@@ -0,0 +1,18 @@
+name: Check markdown links
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  markdown-link-check:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - uses: gaurav-nelson/github-action-markdown-link-check@v1
```

Comment:
do we need to provide the base-branch: https://github.com/gaurav-nelson/github-action-markdown-link-check?tab=readme-ov-file#custom-variables ?

Looks like default is `master`, but we use `main`?

Reply:
Good catch, I missed that.